src/bin/index.ts

```diff
@@ -6,14 +6,16 @@ import type { CliCommandResponse } from "../types/cli";
 import { awsCredentialProviderChain } from "./aws-credential-provider";
 import { accountReadinessCommand } from "./commands/account-readiness";
 import { awsCdkVersionCommand } from "./commands/aws-cdk-version";
-import { bootstrap } from "./commands/bootstrap";
+import { bootstrapCommand } from "./commands/bootstrap";
 import { checkPackageJson } from "./commands/check-package-json";
+import { newCdk } from "./commands/newCdk";
 
 const Commands = {
   AwsCdkVersion: "aws-cdk-version",
   AccountReadiness: "account-readiness",
   CheckPackageJson: "check-package-json",
   New: "new",
+  Bootstrap: "bootstrap",
```

Comment:
Minor: I would probably name this `bootstrap-account` or `prepare-account`, just to avoid people confusing it with the `new` command, which also uses 'bootstrap' terminology: https://github.com/guardian/cdk/blob/129aa9e37c6b8b46855b5b5f775368d476052dec/src/bin/commands/bootstrap.ts#L17-L23

Reply:
Ah yep, my strategy was to rename the bootstrap terminology within the new command but this obviously got missed! Will update this too. (to `NewCdkProps`).

Reply:
Heads up: I'd like to further rename `NewCdkProps` (and all related things) to `NewCdkProjectProps`. This should be a different PR and will include other changes to the `new` command such as removing the `multi-app` and `init` flags that (IMO) are not providing much value.

In short, I think the `new` should be simplified by assuming one hasn't setup `cdk` in their repository, and the repository holds a single stack.